src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -9,15 +9,15 @@
 
 namespace Microsoft.ML
 {
-    using LROptions = LogisticRegression.Options;
+    using LROptions = LogisticRegressionBinaryClassificationTrainer.Options;
 
     /// <summary>
     /// TrainerEstimator extension methods.
     /// </summary>
     public static class StandardTrainersCatalog
     {
         /// <summary>
-        /// Predict a target using a linear classification model trained with <see cref="SgdBinaryTrainer"/>.
+        /// Predict a target using a linear classification model trained with <see cref="SgdCalibratedTrainer"/>.
```

Comment:
Here's a place where the acronym `Sgd` isn't expanded out in the summary comments.